webpack.config.js

```diff
@@ -41,9 +41,6 @@ module.exports = {
       }
     ]
   },
-  externals: {
```

Comment:
sorry I cannot accept this PR, removing this entirely makes webpack bundling jquery library inside the plugin dist js file.

Read Webpack docs https://webpack.js.org/configuration/externals/

And please check this solution: https://github.com/webpack/webpack/issues/4258#issuecomment-340240162

I think this is the way to go:

```js
new webpack.ProvidePlugin({
  $: 'jquery',
  jQuery: 'jquery'
});
```

Then the src code may need some adjustements.